### DISCUSSION TITLE
reaction emoji skin tone

### DISCUSSION THREAD

**Discussion by ghost** [community/community#19299] [Other Feature Feedback, Questions, & Ideas]
when adding a reaction on GitHub, the user's skin tone preference is not used. it would be nice if there was a way to use that skin tone, or to allow skin tone selection when reacting

**Comment by verduscos** [comment]
Great idea! I do love that slack and others offer this option.

**Comment by MacioSzekTV** [comment]
Nice idea, but what do you need skin tone for?

**Reply by ghost** [reply]
for inclusivity

**Comment by queenofcorgis** [comment]
Thanks for adding this - I've added labelling to this comment so it can find its way to the proper team 😄
